environments.json

```diff
@@ -48,7 +48,7 @@
     ]
   },
   "closure": {
-    "full": "Closure Compiler v20180204",
```

Comment:
is there no value in keeping results from older versions?

Reply:
@ljharb Closure Compiler is updated once every month or two month.
If this project can store the all releases, it's great and I would update this PR.

Reply:
That seems ideal to me (assuming the old ones can be marked as obsolete/very as appropriate, so the UI stays manageable) - let's see what other collabs think.

Reply:
This is separated column version: https://github.com/teppeis/compat-table/tree/closure-compiler-v20180319-new-column